src/screens/TopicMemberScreen/index.js

```diff
@@ -31,13 +23,13 @@ const styles = StyleSheet.create({
     backgroundColor: colors.white
   },
   fragmentContentContainer: {
-    flexGrow: 1
+    // flexGrow: 1,
+    backgroundColor: 'white'
   }
 });
 
 const TopicMemberScreen = () => {
   const route = useRoute();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `flexGrow: 1` style property has been commented out. This could affect the layout of the component. Ensure that this change does not negatively impact the UI.
```diff
-    flexGrow: 1
+    // flexGrow: 1,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->